src/language.jsx

```diff
@@ -0,0 +1,47 @@
+import { isCommandMode } from './notebook-utils'
+import { addTask, dispatcher } from './task-definitions'
+
+const languages = []
+const langByName = {}
```

Comment:
Sure.  Makes sense.  One thing about the jsmd parsing though -- we need a way to read in a jsmd before knowing what languages will be loaded, since, at least how I envision this, languages will be added by adding an external dependency near the top of the notebook (which would all happen after jsmd parsing, obviously).